model-mesh/README.md

```diff
@@ -32,3 +32,41 @@ Example ServingRuntime and Predictors can be found at:  https://github.com/kserv
 ### Overlays
 
 None
+
+### Installation process
+
+Following are the steps to install Model Mesh as a part of OpenDataHub install:
+
+1. Install the OpenDataHub operator
+2. Create a KfDef that includes the model-mesh component with the odh-model-controller overlay.
+   
+```
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: opendatahub
+  namespace: opendatahub
+spec:
+  applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: odh-common
+      name: odh-common
+    - kustomizeConfig:
+        overlays:
+          - odh-model-controller
+        repoRef:
+          name: manifests
+          path: model-mesh
+      name: model-mesh
+  repos:
+    - name: kf-manifests
```

Comment:
I'd say we don't need to include `kf-manifests` in this kfdef.

Reply:
Done, took the `kf-manifests` and the corresponding uri out.